src/iostream.h

```diff
@@ -13,7 +13,7 @@
 #ifndef GAP_IOSTREAM_H
 #define GAP_IOSTREAM_H
 
-
+extern Int SIGCHLDHandlerDisabled;
```

Comment:
Please a comment to this which explains what's it about.

In particular, one might think this can be used to temporarily turn of SIGCHLD handling, and turn it back on later. But that's not true, as toggling the variable does not really toggle the actual state of things -- i.e. turning it "off" does not remove an already set signal handler, and turning it "on" does not set one.

One could add an API to do that properly, but I realize you are just trying to get minimal changes to enable "libgap" in. Fair enough. So I am just asking for a clarifying comment ;-).